Update Luangthongkum-2019-341.tsv

MuffinLinwist opened the pull request:
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [x] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

This is the new mapping for Luangthongkumkaren concept list.

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -28,13 +28,13 @@ Luangthongkum-2019-341-26	26	blow (a fire)	176	BLOW (WITH MOUTH)
 Luangthongkum-2019-341-28	28	boat	1844	BOAT
 Luangthongkum-2019-341-29	29	body dirt		
-Luangthongkum-2019-341-30	30	boil (v.)	2493	BOIL
+Luangthongkum-2019-341-30	30	boil (v.)	2492	BOIL (SOMETHING)
```
Its not specified whether it is the transitive or intransitive meaning, so we should stay with the broader mapping BOIL

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -28,13 +28,13 @@ Luangthongkum-2019-341-26	26	blow (a fire)	176	BLOW (WITH MOUTH)
 Luangthongkum-2019-341-33	33	brain	1583	BRAIN
-Luangthongkum-2019-341-34	34	bran,chaff		
+Luangthongkum-2019-341-34	34	bran,chaff	2362	BRAN
 Luangthongkum-2019-341-35	35	breasts	1402	BREAST
-Luangthongkum-2019-341-36	36	broken, split, separate		
+Luangthongkum-2019-341-36	36	broken, split, separate	588	BROKEN
```
I'd leave both unmapped

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -58,15 +58,15 @@ Luangthongkum-2019-341-56	56	classifier for plants, e.g. bamboo, rice, banana et
 Luangthongkum-2019-341-58	58	classifier for long objects		
 Luangthongkum-2019-341-59	59	classifier for four-legged animals		
-Luangthongkum-2019-341-60	60	close (one’s eyes)	2063	CLOSE (EYES)
+Luangthongkum-2019-341-60	60	close (one’s eyes)	2211	CLOSE THE EYES
```
[CLOSE THE EYES](https://concepticon.clld.org/parameters/2211) seems to be discontinued, so the mapping should remain

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -58,15 +58,15 @@ Luangthongkum-2019-341-56	56	classifier for plants, e.g. bamboo, rice, banana et
 Luangthongkum-2019-341-66	66	crocodile	1857	CROCODILE
 Luangthongkum-2019-341-67	67	crop (of fowl), gizzard		
-Luangthongkum-2019-341-68	68	cross (v.)	574	TRANSIT
+Luangthongkum-2019-341-68	68	cross (v.)		
```
"cross" is usually mapped to TRANSIT, so this should stay

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -75,24 +75,24 @@ Luangthongkum-2019-341-73	73	dark, late evening
 Luangthongkum-2019-341-75	75	day	1225	DAY (NOT NIGHT)
 Luangthongkum-2019-341-76	76	dead,die	2101	BE DEAD OR DIE
-Luangthongkum-2019-341-77	77	deaf	996	DEAF
+Luangthongkum-2019-341-77	77	deaf	458	DEAF PERSON
```
not remap

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -75,24 +75,24 @@ Luangthongkum-2019-341-73	73	dark, late evening
+Luangthongkum-2019-341-77	77	deaf	458	DEAF PERSON
 Luangthongkum-2019-341-78	78	deep	1593	DEEP
-Luangthongkum-2019-341-79	79	deer (barking~)	3152	MUNTJACS
+Luangthongkum-2019-341-79	79	deer (barking~)		
```
not remap, barking deers seem to be a thing

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -75,24 +75,24 @@ Luangthongkum-2019-341-73	73	dark, late evening
 Luangthongkum-2019-341-90	90	dream (v.)	1920	DREAM (SOMETHING)
 Luangthongkum-2019-341-91	91	drink(v.)	1401	DRINK
-Luangthongkum-2019-341-92	92	dry(v.)		
+Luangthongkum-2019-341-92	92	dry(v.)	2015	MAKE DRY
```
transitivity unclear, not map

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -334,8 +334,8 @@ Luangthongkum-2019-341-332	332	wife	1199	WIFE
-Luangthongkum-2019-341-336	336	wrong, mistake, error		
-Luangthongkum-2019-341-337	337	yam, potato		
+Luangthongkum-2019-341-336	336	wrong, mistake, error	1390	WRONG
+Luangthongkum-2019-341-337	337	yam, potato	410	YAM
```
different meanings, rather not remap

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -334,8 +334,8 @@ Luangthongkum-2019-341-332	332	wife	1199	WIFE
 Luangthongkum-2019-341-335	335	wing	1257	WING
-Luangthongkum-2019-341-336	336	wrong, mistake, error		
-Luangthongkum-2019-341-337	337	yam, potato		
+Luangthongkum-2019-341-336	336	wrong, mistake, error	1390	WRONG
```
different meanings, rather not remap

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -320,11 +320,11 @@ Luangthongkum-2019-341-318	318	waist	1681	WAIST
 Luangthongkum-2019-341-320	320	warm	1232	WARM
 Luangthongkum-2019-341-321	321	wash (the face)		
-Luangthongkum-2019-341-322	322	wash (clothes)	2503	LAUNDER (WASH CLOTHES)
+Luangthongkum-2019-341-322	322	wash (clothes)	3434	WASH (CLOTHES)
```
WASH (CLOTHES) is not used, so we'd rather stay with LAUNDER

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -309,7 +309,7 @@ Luangthongkum-2019-341-307	307	tongue	1205	TONGUE
 Luangthongkum-2019-341-309	309	tortoise	895	TORTOISE
 Luangthongkum-2019-341-310	310	tread	575	TREAD
-Luangthongkum-2019-341-311	311	trunk (of a tree), firewood		
+Luangthongkum-2019-341-311	311	trunk (of a tree), firewood	344	TREE TRUNK
```
different meanings, rather no mapping

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -281,7 +281,7 @@ Luangthongkum-2019-341-279	279	steal (v.)	713	STEAL
 Luangthongkum-2019-341-281	281	stomach	819	STOMACH
 Luangthongkum-2019-341-282	282	stone, rock	2125	STONE OR ROCK
-Luangthongkum-2019-341-283	283	string, rope	1218	ROPE
+Luangthongkum-2019-341-283	283	string, rope	362	STRING
```
different meanings, rather no mapping

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -134,7 +134,7 @@ Luangthongkum-2019-341-132	132	grasshopper	607	GRASSHOPPER
 Luangthongkum-2019-341-134	134	great-grandchild		
 Luangthongkum-2019-341-135	135	great-great-gandchild		
-Luangthongkum-2019-341-136	136	hair (head~)	2648	HAIR (HEAD)
+Luangthongkum-2019-341-136	136	hair (head~)	1040	HAIR
```
head is specified, mapping should remain

Tarotis reviewed (changes requested):
I was quite surprised to see that the list was already there, but then again, the mappings did need improvement, so thanks a lot for fixing this, there are some really good and important catches.

There is one general trend I am observing: If we have two rather different meanings present, like "ashes" and "fireplace", or "eye", "face", the entry should not be mapped. In other cases, it's important to pay attention to the details in concepticon: Are there two similar concepts, of which one is not used actively in concepticon (launder vs wash clothes)? What is the transitivity of the action?

I did not go through all changes, but especially the first pattern is quite frequent. It would be great if you could go through the changes again and check for the three cases I just described. I expect that there are a lot less changes necessary than have actually been implemented right now.

LinguList commented:
I agree with @tarotis, thanks for checking this. If these suggestions are accounted for, you can merge.

MuffinLinwist commented:
@Tarotis, I have implemented the changes and you requested me. Also, I identified some other entries with the first pattern you mentioned so I modified them as well. Now, the concept list should be optimal.

LinguList commented:
@Tarotis, can I ask you to merge etc. if you find all is done well?

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -84,15 +84,15 @@ Luangthongkum-2019-341-82	82	descent, go down, set	2014	GO DOWN (DESCEND)
 Luangthongkum-2019-341-84	84	dip up		
 Luangthongkum-2019-341-85	85	Dipterocarpus tuberculatus Roxb		
-Luangthongkum-2019-341-86	86	disappear, lost		
+Luangthongkum-2019-341-86	86	disappear, lost	485	DISAPPEAR
```
"disappear" and "lost" are two different meanings, would not map

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -84,15 +84,15 @@ Luangthongkum-2019-341-82	82	descent, go down, set	2014	GO DOWN (DESCEND)
 Luangthongkum-2019-341-89	89	dove	1853	DOVE
 Luangthongkum-2019-341-90	90	dream (v.)	1920	DREAM (SOMETHING)
 Luangthongkum-2019-341-91	91	drink(v.)	1401	DRINK
 Luangthongkum-2019-341-92	92	dry(v.)		
```
Could be mapped to DRY UP

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -84,15 +84,15 @@ Luangthongkum-2019-341-82	82	descent, go down, set	2014	GO DOWN (DESCEND)
 Luangthongkum-2019-341-92	92	dry(v.)		
 Luangthongkum-2019-341-93	93	drunk	1463	DRUNK
-Luangthongkum-2019-341-94	94	dung, excrement	675	FAECES (EXCREMENT)
+Luangthongkum-2019-341-94	94	dung, excrement	2057	DUNG (ANIMAL FAECES)
```
would not map, seems to refer to both faeces (human) and dung (animals)

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -106,12 +106,12 @@ Luangthongkum-2019-341-104	104	enter	749	ENTER
 Luangthongkum-2019-341-106	106	extinguish (t.v.)	127	EXTINGUISH
 Luangthongkum-2019-341-107	107	eye, face		
-Luangthongkum-2019-341-108	108	fall down	1280	FALL
+Luangthongkum-2019-341-108	108	fall down	2894	TUMBLE (FALL DOWN)
```
most previous cases of "fall down" are mapped to FALL, so I would stick with this

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -161,19 +161,19 @@ Luangthongkum-2019-341-159	159	leech (land~)	2273	LEECH
 Luangthongkum-2019-341-161	161	left (side)	244	LEFT
 Luangthongkum-2019-341-162	162	leg	1297	LEG
-Luangthongkum-2019-341-163	163	lemur (flying~), flying squirrel		
+Luangthongkum-2019-341-163	163	lemur (flying~), flying squirrel	3271	FLYING SQUIRREL
```
would not map, flying lemurs and flying squirrels are different species

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -161,19 +161,19 @@ Luangthongkum-2019-341-159	159	leech (land~)	2273	LEECH
 Luangthongkum-2019-341-170	170	linear measure of about two yards		
 Luangthongkum-2019-341-171	171	listen to	1608	LISTEN
-Luangthongkum-2019-341-172	172	live, stay		
+Luangthongkum-2019-341-172	172	live, stay	363	DWELL (LIVE, RESIDE)
```
difficult to interpret "stay" in this context, would probably keep unmapped

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -161,19 +161,19 @@ Luangthongkum-2019-341-159	159	leech (land~)	2273	LEECH
 Luangthongkum-2019-341-173	173	liver	1224	LIVER
 Luangthongkum-2019-341-174	174	long (of time)	418	FOR A LONG TIME
-Luangthongkum-2019-341-175	175	long (of length), tall, high	2143	LONG OR TALL
+Luangthongkum-2019-341-175	175	long (of length), tall, high	1203	LONG
```
should stay LONG OR TALL, otherwise we lose the meanings of "tall" and "high"

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -195,7 +195,7 @@ Luangthongkum-2019-341-193	193	moon, month	2463	MOON OR MONTH
 Luangthongkum-2019-341-195	195	mouse, rat	2139	MUROID (MOUSE OR RAT)
 Luangthongkum-2019-341-196	196	mynah		
-Luangthongkum-2019-341-197	197	nail		
+Luangthongkum-2019-341-197	197	nail	2128	CLAW OR NAIL
```
Are we sure there is some CLAW meaning to it? Otherwise, it could be FINGERNAL OR TOENAIL, which is what "nail" is usually mapped to

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -207,41 +207,41 @@ Luangthongkum-2019-341-205	205	old (of humans)	2112	OLD (AGED)
 Luangthongkum-2019-341-207	207	one	1493	ONE
 Luangthongkum-2019-341-208	208	open (the mouth)	3437	OPEN (MOUTH)
-Luangthongkum-2019-341-209	209	ox, bull, cow	3044	BOVINE
+Luangthongkum-2019-341-209	209	ox, bull, cow		
```
could be mapped to CATTLE I think

Tarotis commented on concepticondata/conceptlists/Luangthongkum-2019-341.tsv:
```diff
@@ -254,7 +254,7 @@ Luangthongkum-2019-341-252	252	sharp	1396	SHARP
 Luangthongkum-2019-341-254	254	shivering	52	SHIVER
 Luangthongkum-2019-341-255	255	shoot (v.)	1172	SHOOT
-Luangthongkum-2019-341-256	256	short, low	2484	SMALL (NOT TALL)
+Luangthongkum-2019-341-256	256	short, low	1645	SHORT
```
"short" and "low" are different meanings, would not map to one of them

Tarotis reviewed (changes requested):
Thanks for the update @MuffinLinwist . I added now a full review. Once these changes are in, we can merge.

MuffinLinwist commented:
@Tarotis, thanks for the full review. Now all the changes are implemented and the checks have been passed.

Tarotis commented:
Thanks, this looks good now, thank you.
